whylog_vim/output_formater/consts.py

```diff
@@ -0,0 +1,5 @@
+class LogReader(object):
+    ITEM = 'investigation item'
+    LINE_HEADER = '--- %s [%s offset %s]:'
+    EMPTY_OUTPUT = 'There is no cause of this line in config.'
+    EMPTY_OUTPUT2 = 'To add new rules press whylog teach <F4> by default.'
```

Comment:
I suggest: 'To add new rules press whylog teach key (<F4> by default).'
Or to know that "whylog teach" is a name: 'To add new rules press WhylogTeach key (<F4> by default).'